pyproject.toml

```diff
@@ -28,6 +28,8 @@ flake8-variables-names = "^0.0.5"
 flake8-builtins = "^2.1.0"
 pep8-naming = "^0.13.3"
 pre-commit = "^3.2.1"
+pytest-freezegun = "^0.4.2"
```

Comment:
Since you manage all requests and responses from fakeAPI, and you know what dates it returns, this library looks redundant

Reply:
in this test_huntflow_token_proxy.test_update_token__ok, I want to show that the token update method updates the token correctly using the refresh_data. I also want to make sure that the correct token expiration time is set. But since the token update method sets the expiration date using time.time(), you will have to use a freezer to further compare the dates in the test